frontend/src/component/playground/Playground/AdvancedPlayground.tsx

```diff
@@ -304,11 +354,16 @@ export const AdvancedPlayground: VFC<{
                                         Object.values(errors).length === 0
                                     }
                                     show={
-                                        <AdvancedPlaygroundResultsTable
-                                            loading={loading}
-                                            features={results?.features}
-                                            input={results?.input}
-                                        />
+                                        <>
+                                            <GenerateWarningMessages
+                                                response={results}
+                                            />
```

Comment:
This component will be `null` when no warning, I'm just wondering if this is ok (i.e. react knows how to handle null components). Even if react supports this, I remember we were using conditional rendering rather than returning null.

Reply:
I'm pretty sure that react knows how to handle nulls, yeah. I think it's even preferable to, say, a, fragment. `ConditionallyRender` also returns `null` if it's not rendering anything.

Reply:
Sounds reasonable! I kind of like this approach because it requires less boilerplate code (I mean, tags are boilerplate in a way). So I think it's good and something for my TIL list :)